testsuite/ui/views/devel/__init__.py

```diff
@@ -115,3 +121,23 @@ def signup(self, org: str, username: str = None, email: str = None, submit: bool
     def is_displayed(self):
         return BaseDevelView.is_displayed.fget(self) and self.path in self.browser.url and \
                self.organization.is_displayed and self.username.is_displayed and self.email.is_displayed
+
+
+class DocsView(BaseDevelView):
+    """View for Documentation page of devel portal"""
+    path_pattern = "/docs"
+    active_docs_section = ActiveDocV3Section()
+
+    def __init__(self, parent, path=None):
+        super().__init__(parent)
+        if path:
```

Comment:
In your current implementation this is redundant. `path` is constantly rewritten by `BaseDevelView` to `/docs`. Please add if statement: `if not self.path:` to the base view constructor